newIDE/app/src/UI/DragAndDrop/DragSourceAndDropTarget.js

```diff
@@ -43,6 +44,12 @@ export const makeDragSourceAndDropTarget = <DraggedItemType>(
   reactDndType: string
 ): ((Props<DraggedItemType>) => React.Node) => {
   const sourceSpec = {
+    canDrag(props: Props<DraggedItemType>, monitor: DragSourceMonitor) {
+      const item = monitor.getItem();
+      const canDrag = props.canDrag || null;
```

Comment:
nitpicking: avoid null as much as possible - keep your variables to a simple type (string/boolean/number) if you don't need more.
```suggestion
      const canDrag = props.canDrag || false;
```